File: src/lessons/lander/lvl2_locate_landing_zone/LocateLandingZone.scala
package lessons.lander.lvl2_locate_landing_zone

import plm.universe.lander._
import plm.universe.lander.Configurations._
import plm.core.model.Game
import plm.core.model.lesson.ExerciseTemplated
import plm.core.model.lesson.Lesson

class LocateLandingZone(game: Game, lesson: Lesson) extends ExerciseTemplated(game, lesson, null) {
  tabName = "Lander"
  setup(Array(
      makeWorld(game, "CHALLENGING_TERRAIN_SIMPLE_CONFIG"),
      makeWorld(game, "SIMPLE_TERRAIN_TRIVIAL_CONFIG")))
}

File: src/lessons/lander/lvl2_locate_landing_zone/LocateLandingZone.java
package lessons.lander.lvl2_locate_landing_zone;

import static lessons.lander.universe.Configurations.CHALLENGING_TERRAIN_SIMPLE_CONFIG;
import static lessons.lander.universe.Configurations.SIMPLE_TERRAIN_TRIVIAL_CONFIG;

import lessons.lander.universe.LanderWorld;
import plm.core.model.lesson.ExerciseTemplated;
import plm.core.model.lesson.Lesson;

public class LocateLandingZone extends ExerciseTemplated {
  public LocateLandingZone(Lesson lesson)
  {
    super(lesson, null);
    tabName = "Lander";
    setup(new LanderWorld[] {CHALLENGING_TERRAIN_SIMPLE_CONFIG, SIMPLE_TERRAIN_TRIVIAL_CONFIG});
  }
}
